.github/workflows/tsc.yml

```diff
@@ -0,0 +1,16 @@
+name: tsc
```

Comment:
```suggestion
name: Type Check
```

How about using other descriptions like *Type Check* or better one, not command name *tsc*. It's just suggestion 😉